frontend/src/components/YesNoSelection.tsx

```diff
@@ -62,37 +69,20 @@ color: ${CORAL};
 
 const YesNoSelection: FunctionComponent<YesNoSelectionProps> = ({correctAnswer}): ReactElement => {
   let [selected, setSelected] = useState(-1);
-  const answer = correctAnswer ? 1 : 0;
+
+  const select = (num: number) => {
+    selected === -1 && setSelected(num)
+  }
 
   return (
     <Blocker afterSeconds={15} repeatable={false} message='Yes or No?'>
       <YesNoContainer>
-      {selected === answer ?
-      <>
-        <YesNoButton highlight={1} onClick={() => setSelected(1)} text="yes" top={20}
+      <YesNoButton highlight={selected === 1 ? correctAnswer ? 1 : 0 : -1} onClick={() => select(1)} text="yes" top={20}
                          check={<CheckCorrect/>}
                          icon={<LikeFilled/>}/>
-            <YesNoButton highlight={0} onClick={() => setSelected(0)} text="no" top={20}
-                         check = {<CheckIncorrect/>}
-                         icon={<DislikeFilled/>}/>
-      </> : 
-      selected === -1 ?
-              <>
-                <YesNoButton highlight={-1} onClick={() => setSelected(1)} text="yes" top={20}
-                             icon={<LikeFilled/>}/>
-                <YesNoButton highlight={-1} onClick={() => setSelected(0)} text="no" top={20}
-                             icon={<DislikeFilled/>}/>
-
-              </>
-              :
-              <>
-                <YesNoButton highlight={1} onClick={() => setSelected(1)} text="yes" top={20}
-                             check = {<CheckCorrect/>}
-                             icon={<LikeFilled/>}/>
-                <YesNoButton highlight={0} onClick={() => setSelected(0)} text="no" top={20}
-                             check = {<CheckIncorrect/>}
-                             icon={<DislikeFilled/>}/>
-              </>}
+      <YesNoButton highlight={selected === 0 ? correctAnswer ? 0 : 1 : -1} onClick={() => select(0)} text="no" top={20}
```

Comment:
so hmm the expected behavior would be: the kid should be able to select either Yes Or No, as in like they can still change their answer after picking one. If they answer it right, then you have the feedback icon (which is either a check mark or an X) once they select an answer. You can check the figma file. I think right now you are only allowed to select one and you can't change it